### DISCUSSION TITLE
Updating react to v18, possible documentation in the future?

### DISCUSSION THREAD

**Discussion by RookTKO** [shakacode/react_on_rails#1474] [Q&A] [answered]
I know theres several moving pieces when working with shakacode react_on_rails and react and webpacker (etc) but I was hoping it would be possible to get some kind of step by step tutorial on what I would need to update for shakacode for my application to work. I currently started the process and update my react to 18 and am already getting issues and trying to traverse what needs to be updated. So will there be a guide for this in the future possibly?

**Comment by Judahmeek** [comment, marked as answer]
@RookTKO sorry for the late reply.

The biggest upgrade you mention is obviously webpacker => shakapacker, so https://github.com/shakacode/shakapacker/blob/master/CHANGELOG.md is most likely to document the changes that are necessary.

It's hard to make a step-by-step guide though, due to all the possible custom configuration.
